src/api/nodeRpcCall.js

```diff
@@ -2034,19 +2077,19 @@ const getScheduledCalls = async () => {
       .map((item) => ({
         ...item,
         call: item.call.unwrap(),
-      })))
-    .filter((item) => item.call.call.isLookup // our FE only does lookups now
-      && item.call.maybePeriodic.isNone); // we're interested only in referendum results, so nonperiodic
+      })));
 
+  const lookupItems = agendaItems.filter((item) => item.call.call.isLookup // our FE only does lookups now
+      && item.call.maybePeriodic.isNone); // we're interested only in referendum results, so nonperiodic
   // we only want do download small preimages. fetching multi-megabyte setCode could be painful.
```

Comment:
Maybe we should make a note and display to the user if there are lookupitems that are bigger, but just with a link to polkadotjs where he can see it. Is it possible to get something around the users if a call is purposefully big ?